src/index.html

```diff
@@ -11,7 +11,144 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fs">
+        <legend>Personal information</legend>
+        <div class="formfield">
+          <label for="surname">
+                  Surname: <input type="text" required
+name="surname" id="surname">
+          </label>
```

Comment:
Fix in all cases

[checklist p3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

```suggestion
          <label for="surname">
            Surname:
            <input
              type="text"
              required
              name="surname"
              id="surname"
            >
          </label>
```